components/title/multisize/src/index.js

```diff
@@ -0,0 +1,33 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const TitleMultisize = ({className, postTitle, preTitle, title}) => (
+  <div className={cx('sui-TitleMultisize', className)}>
+    {preTitle && <h3 className='sui-TitleMultisize-preTitle'>{preTitle}</h3>}
+    {title && <h1 className='sui-TitleMultisize-title'>{title}</h1>}
+    {postTitle && <h3 className='sui-TitleMultisize-postTitle'>{postTitle}</h3>}
+  </div>
+)
+
+TitleMultisize.displayName = 'TitleMultisize'
+
+TitleMultisize.propTypes = {
+  /**
+   * CSS classNames to apply to Title Multisize container.
+   */
+  className: PropTypes.node,
+  /**
+   * Text to display after main title (h3).
+   */
+  postTitle: PropTypes.node,
```

Comment:
Yep, this is an array of strings and elements, but this is exactly what PropTypes.node is:

PropTypes.node
// Anything that can be rendered: numbers, strings, elements or an array
// (or fragment) containing these types.